build.sbt

```diff
@@ -128,12 +130,23 @@ lazy val root = Project("ctd-omega-editorial-frontend", file("."))
       MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
     ),
     headerLicense := Some(HeaderLicense.MIT("2022", "The National Archives")),
+    resolvers += Resolver.githubPackages("rwalpole"),
     libraryDependencies ++= Seq(
       guice,
-      "org.webjars.npm"         % "govuk-frontend"     % "4.3.1",
-      "uk.gov.hmrc"            %% "play-frontend-hmrc" % "6.2.0-play-28",
-      "org.scalatestplus.play" %% "scalatestplus-play" % "5.0.0"  % Test,
-      "org.jsoup"               % "jsoup"              % "1.15.3" % Test
+      "dev.fpinbo"             %% "jms4s-simple-queue-service" % "0.0.1-53518bb-SNAPSHOT",
+      "org.typelevel"          %% "log4cats-core"              % "2.5.0",
+      "org.typelevel"          %% "log4cats-slf4j"             % "2.5.0",
+      "org.typelevel"          %% "cats-core"                  % "2.9.0",
+      "org.typelevel"          %% "cats-effect"                % "3.4.4",
+      "org.typelevel"          %% "cats-effect-kernel"         % "3.4.4",
+      "org.webjars.npm"         % "govuk-frontend"             % "4.3.1",
+      "uk.gov.hmrc"            %% "play-frontend-hmrc"         % "6.2.0-play-28",
+      "org.scalatestplus.play" %% "scalatestplus-play"         % "5.0.0"  % Test,
+      "org.jsoup"               % "jsoup"                      % "1.15.3" % Test
+    )
+    .map(_.exclude("org.slf4j", "*")),
+    libraryDependencies ++= Seq(
+      "ch.qos.logback" % "logback-classic" % "1.3.0"
```

Comment:
This should be added so that `sbt` is aware of the token:

```
githubTokenSource := TokenSource.Or(
  TokenSource.Environment("GITHUB_TOKEN"),
  TokenSource.GitConfig("github.token") //  ~/.gitconfig
)
```

Reply:
Confirmed.